packages/plugin-js/test/build.test.js

```diff
@@ -19,9 +19,9 @@ describe('@cobalt-ui/plugin-js', () => {
         ],
         color: {},
       });
-      expect(fs.readFileSync(new URL('actual.js', cwd), 'utf8'), `${dir}: JS`).toBe(fs.readFileSync(new URL('want.js', cwd), 'utf8'));
-      expect(fs.readFileSync(new URL('actual.d.ts', cwd), 'utf8'), `${dir}: TS`).toBe(fs.readFileSync(new URL('want.d.ts', cwd), 'utf8'));
-      expect(fs.readFileSync(new URL('actual.json', cwd), 'utf8'), `${dir}: JSON`).toBe(fs.readFileSync(new URL('want.json', cwd), 'utf8'));
+      expect(fs.readFileSync(new URL('actual.js', cwd), 'utf8'), `${dir}: JS`).toMatchFileSnapshot(fileURLToPath(new URL('want.js', cwd)));
```

Comment:
Thank god for `toMatchFileSnapshot()` I thought I’d have to normalize the line endings 😓 

🤔 I really need to use this more often